src/main/java/net/greatstart/controllers/EventController.java

```diff
@@ -17,19 +14,5 @@ public EventController(EventService eventService) {
         this.eventService = eventService;
     }
 
-    public Event saveEvent(Event event) {
-        return eventService.saveEvent(event);
-    }
-
-    public void deleteEvent(long id) {
-        eventService.deleteEvent(id);
-    }
-
-    public Event getEventbyId(long id) {
-        return eventService.getEventById(id);
-    }
-
-    public List<Event> getAllEvents() {
-        return eventService.getAllEvents();
-    }
+    //TODO: to be implemented
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)